packages/plugin-utils/src/processing/processFill.ts

```diff
@@ -0,0 +1,219 @@
+import type CreativeEditorSDK from '@cesdk/cesdk-js';
+import { type Source } from '@cesdk/cesdk-js';
+import type FillProcessingMetadata from '../metadata/FillProcessingMetadata';
+import { Optional } from '../types/Optional';
+
+export default async function processFill<T>(
+  cesdk: CreativeEditorSDK,
+  blockId: number,
+  metadata: FillProcessingMetadata,
+  process: (
+    sources: Optional<Source, 'width' | 'height'>[],
```

Comment:
I'm not quite sure what `sources` exactly mean

Reply:
https://img.ly/docs/cesdk/engine/guides/source-sets/

Reply:
I see, so `sources` is a  `sourceSet`? Would it maybe make more sense to call it `SourceSet` so it's discoverable?

Reply:
Done